src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -152,37 +155,14 @@
             </ul>
           </div>
 
-          <!-- the details section -->
-          <div class="comments-section mt-8" v-if="item.comments.length > 0">
-            <v-divider></v-divider>
-            <div class="title-bar mt-5">
-              <h4>Detail{{item.comments.length > 1 ? "s" : ""}} ({{item.comments.length}})</h4>
-              <v-btn
-                color="primary"
-                v-if="isRoleStaff"
-                @click.stop="showCommentDialog(item.filingId)"
-              >
-                <span>Add Detail</span>
-              </v-btn>
-            </div>
-            <div>
-              <!-- the details list-->
-              <v-list>
-                <v-list-item class="pl-0 pr-0" v-for="(comment, index) in item.comments" :key="index">
-                  <v-list-item-content>
-                    <v-list-item-title class="body-2">
-                      <strong v-if="!isRoleStaff">Registry Staff</strong>
-                      <strong v-else>{{comment.submitterDisplayName || 'N/A'}}</strong>
-                      ({{convertUTCTimeToLocalTime(comment.timestamp)}})
-                    </v-list-item-title>
-                    <v-list-item-subtitle class="body-2">
-                      <div class="pre-line">{{comment.comment}}</div>
-                    </v-list-item-subtitle>
-                  </v-list-item-content>
-                </v-list-item>
-              </v-list>
-            </div>
-          </div>
+          <!-- the detail comments section -->
+          <DetailsList
+            :filing=item
```

Comment:
I'm in favour of renaming "item" to "filing" in the markup.

Reply:
Sure, can update this.